### PR TITLE
ENT-5363 add SUSE 15 to Jenkins

### PR DESCRIPTION
**Pull request by Lex-2008**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

**Comment by Lex-2008** [comment]
green build: https://ci.cfengine.com/job/pr-pipeline/5681/

File: build-scripts/test-on-testmachine [hunk 1]
```diff
@@ -46,6 +46,12 @@ case "$TEST_MACHINE" in
             then
                 continue
             fi
+            # gpg-agent sometimes is spawned on suse during our tests.
+            # This happens only in chroot. Ignore that.
+            if ps -ef | egrep "^ *[^ ]+ +$pid" | grep gpg-agent && grep suse /etc/os-release
```
Comment (vpodzime) [inline build-scripts/test-on-testmachine:51]: A couple things here:

- first test `grep suse /etc/os-release`
- `ps -e -o FIELDS` might be better to get the info we need/want and make parsing easier
- we should still kill those processes, right? `ps -o cmd` should be possible to use to find the *gpg-agent* processes spawned for zypper. `ps --forest` might even be possible to use to find the processes spawned from the tests

**Review by vpodzime — COMMENTED** [review]
Looks good to me otherwise.

File: build-scripts/test-on-testmachine (hunk 1 above)
Comment (Lex-2008) [reply build-scripts/test-on-testmachine]: addressed first two points, regarding the third one - yes, later in the code all processes running in chroot are killed anyway, with a message like this:

```
                echo "Error: Found processes left behind in the chroot. I'm killing them, but the build will fail, so clean up your mess!"
```

**Review by vpodzime — APPROVED** [review]
Looks good to me.

**Review by olehermanse — APPROVED** [review]

**Pull request merged**